### PR TITLE
Update wise descriptions and tooltips

### PR DESCRIPTION
**Pull request by 0xSachinK**
- Add EUR to the landing page carousel
- Update trouble scanning QR code link to direct to pay via desktop
- Update wise descriptions and tooltips
- Update "Add to Chrome" URL to link to Chrome extension
- Update links to relevant pages in docs

### CODE REVIEW

File: client/src/helpers/strings/wise.ts [hunk 1]
```diff
@@ -17,40 +17,37 @@ const strings: PlatformStrings = {
...
-    Provide the transaction alert email you received from Wise containing "You have done a UPI txn"
-    to complete the order. You can sign in with Google to pull the email or manually upload/paste
-    the email.
+    Generate a proof of payment using the ZKP2P Extension to complete the order.
   `,
 
   // New Deposit
   NEW_DEPOSIT_INSTRUCTIONS: `
-    Depositing requires registering a valid Wise multi currency id. Provide the USDC liquidity to deposit and
+    Depositing requires registering a valid Wise multi currency account id. Provide the USDC liquidity to deposit and
```
Comment (asoong) [inline client/src/helpers/strings/wise.ts:40]: Recommend renaming `multi currency id` altogether in the client. All strings that display that currently were development placeholders. Suggest maybe `Wise recipient id`?

File: client/src/helpers/strings/wise.ts [hunk 2]
```diff
@@ -17,40 +17,37 @@ const strings: PlatformStrings = {
...
-    Please follow instructions in the extension to notarize an existing Wise payment in the transactions page.
+    No Wise transfer proofs found.
+    Please follow instructions in the extension to prove an existing Wise payment.
   `,
 
   // New Registration
   REGISTRATION_INSTRUCTIONS: `
-    You must register in order to use ZKP2P. Registration requires a notarized message from Wise, which is used
-    to prove you own a Wise account. Your Wise ID is hashed to conceal your identity.
+    You must register in order to use ZKP2P. Click on Register above to initiate registration.
```
Comment (asoong) [inline client/src/helpers/strings/wise.ts:30]: The second sentence is unnecessary. If there isn't anymore meaningful context we want to share, we can consider removing the component that displays this string altogether and stick it into a tooltip.

File: client/src/helpers/strings/wise.ts [hunk 3]
```diff
@@ -17,40 +17,37 @@ const strings: PlatformStrings = {
...
   NEW_DEPOSIT_ADDITIONAL_REGISTRATION_TOOLTIP: `
-    This is a second registration step required for depositors only to connect a Wise multi currency id
+    This is a second registration step required ONLY for depositors to connect a Wise multi currency account id
     to your account.
   `,
   NEW_DEPOSIT_ID_TOOLTIP: `
     This is a valid Wise tag where users will send payments.
     This connects your Wise tag to your address on chain.
-    This must match the Wise tag you used to register.
+    This Wise tag must be of the account with which you reigstered.
```
Comment (asoong) [inline client/src/helpers/strings/wise.ts:50]: `must be from the account you registered with`

File: client/src/helpers/strings/wise.ts [hunk 4]
```diff
@@ -17,40 +17,37 @@ const strings: PlatformStrings = {
...
 
   // New Deposit
   NEW_DEPOSIT_INSTRUCTIONS: `
-    Depositing requires registering a valid Wise multi currency id. Provide the USDC liquidity to deposit and
+    Depositing requires registering a valid Wise multi currency account id. Provide the USDC liquidity to deposit and
     desired USDC/EUR conversion rate. You will receive EUR payments from users who claim your deposit.
   `,
   NEW_DEPOSIT_ADDITIONAL_REGISTRATION_TOOLTIP: `
-    This is a second registration step required for depositors only to connect a Wise multi currency id
+    This is a second registration step required ONLY for depositors to connect a Wise multi currency account id
```
Comment (asoong) [inline client/src/helpers/strings/wise.ts:44]: - lower `only`
- update mc id to `Wise recipient id`

File: client/src/helpers/strings/wise.ts [hunk 5]
```diff
@@ -71,25 +68,23 @@ const strings: PlatformStrings = {
...
   `,
 
   // Payment Requirements
   PAYMENT_REQUIREMENT_STEP_ONE: `
-    Send payment from your UPI ID linked to your Wise bank account. Do not send payment from UPI Lite or other bank accounts
+    Send payment from your ZKP2P registered Wise acount
   `,
   PAYMENT_REQUIREMENT_STEP_TWO: `
-    Wise InstaAlert email notifications are enabled for your account
+    Chose the right receiving currency (EUR) while sending the payment
```
Comment (asoong) [inline client/src/helpers/strings/wise.ts:82]: - `send and receive currency` perhaps?

File: client/src/helpers/strings/wise.ts [hunk 6]
```diff
@@ -71,25 +68,23 @@ const strings: PlatformStrings = {
...
   `,
   PAYMENT_REQUIREMENT_STEP_THREE: `
-    Send the exact INR amount including decimals as displayed on the payment screen
-  `,
-  PAYMENT_REQUIREMENT_STEP_FOUR: `
-    Lorem ipsum dolor sit ament
+    Send the exact amount including decimals as displayed above
```
Comment (asoong) [inline client/src/helpers/strings/wise.ts:85]: `exact EUR amount`

**Review by asoong — CHANGES_REQUESTED** [review]
Looks good

File: client/src/helpers/strings/wise.ts (hunk 5 above)
Comment (0xSachinK) [reply client/src/helpers/strings/wise.ts:82]: We aren't constraining the send currency, as long as the receiver receives their desired currency and amount.

File: client/src/helpers/strings/wise.ts (hunk 2 above)
Comment (0xSachinK) [reply client/src/helpers/strings/wise.ts:30]: If we remove entirely, this doesn't look very good.
Screenshot 2024-04-08 at 12 15 50 AM

File: client/src/helpers/strings/wise.ts (hunk 5 above)
Comment (asoong) [reply client/src/helpers/strings/wise.ts:82]: We should make this clear somewhere

File: client/src/helpers/strings/wise.ts (hunk 2 above)
Comment (asoong) [reply client/src/helpers/strings/wise.ts:30]: Agreed, I mistook this for the instruction section inside the proving flow. Consider some variation of: `You must register in order to you ZKP2P. Provide proof of a valid Wise account.`

Comment (0xSachinK) [reply client/src/helpers/strings/wise.ts]: Added in 4a5384f

File: client/src/helpers/strings/wise.ts (hunk 5 above)
Comment (0xSachinK) [reply client/src/helpers/strings/wise.ts]: Added in 9468ea2. Although can make it clear somewhere else to make users aware of this flexibility.

File: client/src/helpers/strings/platform.ts [hunk 7]
```diff
@@ -58,13 +59,13 @@ export class PlatformStringProvider {
...
       this.strings = wiseCopy;
-    }  else {
+    } else {
       throw new Error('Invalid platform type');
     }
   }
 
   get(key: keyof PlatformStrings): string {
-    return this.strings[key];
+    return this.strings[key] ?? '';
```
Comment (asoong) [inline client/src/helpers/strings/platform.ts:68]: We should be even louder here i.e. raise an exception if an expected string is not returned

**Review by asoong — APPROVED** [review]
LGTM, one comment

**Review by asoong — COMMENTED** [review]
Minor changes

File: client/src/helpers/strings/wise.ts [hunk 8]
```diff
@@ -71,25 +74,23 @@ const strings: PlatformStrings = {
     Optionally, provide a recipient address below to receive funds in another wallet. Submit transaction to start your order
   `,
   INSTRUCTION_DRAWER_STEP_THREE: `
-    Click 'Send' and complete the payment on any UPI ID linked to your Wise bank account. Ensure you have email notifications from Wise InstaAlerts enabled
+    Click 'Send' and complete the payment from your Wise account. Ensure you have access to a dekstop to prove the payment later
```
[the remark this answers is not in the document]
Comment (richardliang) [reply client/src/helpers/strings/wise.ts:77]: with the QR code, we're actually directing users to a mobile flow. the UX is better if they pay with desktop -> prove desktop

[the remark this answers is not in the document]
Comment (richardliang) [reply client/src/helpers/strings/wise.ts:77]: yeah the verify send and receive are in `PAYMENT_REQUIREMENT_STEP_TWO`

`Choose a send currency and ensure the receive currency is set to the same requested currency above when making the payment.`

**Pull request merged**